manifests/init.pp

```diff
@@ -19,6 +19,20 @@
   Boolean $synchronize_automatically                 = $wsusserver::params::synchronize_automatically,
   String $synchronize_time_of_day                    = $wsusserver::params::synchronize_time_of_day,
   Integer $number_of_synchronizations_per_day        = $wsusserver::params::number_of_synchronizations_per_day,
+  Boolean $send_sync_notification                    = $wsusserver::params::send_sync_notification,
```

Comment:
Can you document these parameters in the README.md ( https://github.com/TraGicCode/tragiccode-wsusserver/blob/master/README.md#wsusserver-1 ) like the other parameters that make up the public interface to this class?

Reply:
Still waiting on you to do this.

Reply:
Done

Reply:
Oh, sorry now i see where you mean

Reply:
Done